src/main/resources/templates/acs-status.ftl

```diff
@@ -23,8 +23,8 @@ ${ACSStatus.checkoutOk}<#rt>
 ${ACSStatus.acsRenewalPolicy}<#rt>
 ${ACSStatus.statusUpdateOk}<#rt>
 ${ACSStatus.offLineOk}<#rt>
-${ACSStatus.timeoutPeriod}<#rt>
-${ACSStatus.retriesAllowed}<#rt>
+${ACSStatus.timeoutPeriod?string("000")}<#rt>
+${ACSStatus.retriesAllowed?string("000")}<#rt>
```

Comment:
I looked at the documentation for "?string" on a number, but it is not clear to me if this truncates the field if it is greater than `999` somehow or what it does if the value is negative. I think these are extreme edge cases at this point and likely in the future.

We might, not necessarily for this PR, want to use `lib.ftl` with this template. There are several cases like this in other SIP responses, so there is a macro `limitNumberToRange` that works like `<@limitNumberToRange value=${ACSStatus.timeoutPeriod} min=0 max=999 length=3/>` which will ensure that the value is within the range and that there is a `0` pad to satisfy the `length` field.